### PR TITLE
Fix containers

### PR DESCRIPTION
**Pull request by jkrocil**
Fix pod navigation

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 843a63ce5ab90cf49548881b8ae6e4ef96b9cd94:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by avi3tal** [comment]
@jkrocil please merge this PR and i'll rebase in my local repo

File: cfme/containers/pod.py [hunk 1]
```diff
@@ -38,12 +39,27 @@ def __init__(self, name, provider):
...
             tb.refresh()
 
+    def get_detail(self, *ident):
+        """ Gets details from the details infoblock
+
+        Args:
+            *ident: An InfoBlock title, followed by the Key name, e.g. "Relationships", "Images"
+        Returns: A string representing the contents of the InfoBlock's value.
+        """
+        self.navigate(detail=True)
```
Comment (mfalesni) [inline cfme/containers/pod.py]: This should rather do ``self.load_details(refresh=True)`` to not navigate the whole path on each query.

**Comment by dajoRH** [comment]
Lint report for commit fcfe860a83037f36a0ce3156b56d9c30a55b5426:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by jkrocil** [comment]
Issue resolved. Please review so that we can merge it and CM QE can continue working on their automation. It can't really be tested anymore, since we don't have an OS provider available, but if there are issues they will be addressed by the CM QE team in their upcoming pull requests. Thank you
@psav @mfalesni

**Comment by mfalesni** [comment]
`/me reviews now ...`

**Comment by mfalesni** [comment]
Seems legit.

**Pull request merged**